### PR TITLE
First js-homework

### PR DESCRIPTION
**Pull request by KsushaDk**
First  JS homework done

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+console.log (Boolean (test));
+
+// как вы думаете, что сейчас в переменной test? Свой вариант напишите в комментарий;
+// true
+
+// / Дан массив [2, 3, 5, 8]. С помощью цикла for найдите произведение элементов этого массива.
+
+let sum = 0;
+let arr = [2, 3, 5, 8];
```
Comment (BubentsovKirill) [inline main.js:43]: Should be a `const`

File: main.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+for (let i = 0; i < arr.length; i++) {
+    sum += arr[i];   
+}
+console.log(sum);
+
+// Дан массив [2, 5, 8, 15, 0, 6, 20, 3]. С помощью цикла for и оператора if 
+// выведите в консоль те элементы массива, которые больше 5-ти, но меньше 10-ти.
+
+let arr = [2, 5, 8, 15, 0, 6, 20, 3];
```
Comment (BubentsovKirill) [inline main.js:52]: Should be a `const`

File: main.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+for (let a = 0; a < arr.length; a++) {
```
Comment (BubentsovKirill) [inline main.js:53]: use 'i' for defining the index of iteartion, it is a standard.

File: main.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+	if(arr[a] > 5 && arr[a] < 10) {
+		console.log(arr[a]);
+	}
+}
+
+// Дан массив [2, 5, 8, 15, 0, 6, 20, 3]. С помощью цикла for и оператора if 
+// выведите в консоль четные элементы массива.
+
+let arr = [2, 5, 8, 15, 0, 6, 20, 3];
```
Comment (BubentsovKirill) [inline main.js:62]: Should be a `const`

File: main.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+for (let a = 0; a < arr.length; a++) {
```
Comment (BubentsovKirill) [inline main.js:63]: use `i`

File: main.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+	if((arr[a] % 2) === 0) {
```
Comment (BubentsovKirill) [inline main.js:64]:
```suggestion
if(!(arr[i] % 2))
```

File: main.js (hunk 1 above)
Comment (KsushaDk) [reply main.js]: fixed

File: main.js (hunk 2 above)
Comment (KsushaDk) [reply main.js]: fixed

File: main.js (hunk 3 above)
Comment (KsushaDk) [reply main.js]: fixed

File: main.js (hunk 4 above)
Comment (KsushaDk) [reply main.js]: fixed

File: main.js (hunk 5 above)
Comment (KsushaDk) [reply main.js]: fixed

File: main.js (hunk 6 above)
Comment (KsushaDk) [reply main.js]: fixed

**Review by KsushaDk — COMMENTED** [review]
Fixed all mistakes

**Pull request merged**